src/plugins/linkAction/LinkAction.js

```diff
@@ -33,10 +33,11 @@ export default class LinkAction {
     }
 
     appliesTo(objectPath) {
-        let domainObject = objectPath[0];
-        let type = domainObject && this.openmct.types.get(domainObject.type);
+        // let domainObject = objectPath[0];
```

Comment:
should we remove the commented out code here?

Reply:
Yep!